src/Utilities/DOM.jl

```diff
@@ -248,7 +248,11 @@ function Base.show(io::IO, n::Node)
         print(io, '<', n.name)
         for (name, value) in n.attributes
             print(io, ' ', name)
-            isempty(value) || print(io, '=', repr(escapehtml(value)))
+            if name === Symbol("data-clipboard-text")
+                isempty(value) || print(io, '=', repr(escapehtml(value, escape_new_lines=true)))
+            else
+                isempty(value) || print(io, '=', repr(escapehtml(value)))
+            end
```

Comment:
`DOM` is an independent generic module, so its behavior shouldn't really depend on arbitrary argument names. I think we can just always escape newlines in attributes? Based on my reading of the standard, having [character references in attribute values should always be fine](https://html.spec.whatwg.org/#attributes-2).